### PR TITLE
[31기 임재혁] Add: Misson 4가지 모두 구현

### PR DESCRIPTION
**Pull request by Jaehyuksssss**
## :: 최근 작업 주제 (하나 이상의 주제를 선택해주세요.)
- [x] 기능 추가
- [ ] 리뷰 반영
- [ ] 리팩토링
- [ ] 버그 수정
- [ ] 컨벤션 수정
<br />
## :: 구현 목표 (해당 브랜치에서 구현하고자 하는 하나의 목표를 설정합니다.)
-리팩토링을 통해서 난잡한 코드를 정리하고 
<br />
### 메인페이지 | Mock data 이용해 여러 피드 구현, 여러 댓글 구현
<br />
## :: 성장 포인트 (해당 기능을 구현하며 고민했던 사항이나 새로 알게된 부분, 어려웠던 점 등을 작성합니다.)
<br />
## :: 기타 질문 및 특이 사항

### CODE REVIEW

File: src/pages/jaehyuk/Login/LoginJaehyuk.js [hunk 1]
```diff
@@ -1,7 +1,90 @@
...
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import './LoginJaehyuk.scss';
 
 function Login() {
-  return <div> test </div>;
+  const navigate = useNavigate();
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [isActive, setIsActive] = useState('deactive');
```
Comment (ghost) [inline src/pages/jaehyuk/Login/LoginJaehyuk.js:9]: __[ESLint]__ 'isActive' is assigned a value but never used. ([view](https://sider.review/gh/repos/470071320/pulls/10#issue-6821391))

<details>

| Rule | Severity | Recommended |
| ---- | -------- | ----------- |
| `no-unused-vars` | `warn` | Yes |

References:
- <https://eslint.org/docs/rules/no-unused-vars>

You can [close this issue](https://sider.review/gh/repos/470071320/pulls/10#issue-6821391) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).

</details>

File: src/pages/jaehyuk/Login/LoginJaehyuk.js [hunk 2]
```diff
@@ -1,7 +1,90 @@
...
+  const [isDisable, setIsDisable] = useState(true);
```
Comment (ghost) [inline src/pages/jaehyuk/Login/LoginJaehyuk.js:10]: __[ESLint]__ 'isDisable' is assigned a value but never used. ([view](https://sider.review/gh/repos/470071320/pulls/10#issue-6821392))

<details>

| Rule | Severity | Recommended |
| ---- | -------- | ----------- |
| `no-unused-vars` | `warn` | Yes |

References:
- <https://eslint.org/docs/rules/no-unused-vars>

You can [close this issue](https://sider.review/gh/repos/470071320/pulls/10#issue-6821392) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).

</details>

File: src/pages/jaehyuk/Login/LoginJaehyuk.js [hunk 3]
```diff
@@ -1,7 +1,90 @@
...
+  const [color, setColor] = useState('#B2DFFC');
+
+  const isPassedLogin = () => {};
```
Comment (ghost) [inline src/pages/jaehyuk/Login/LoginJaehyuk.js:14]: __[ESLint]__ 'isPassedLogin' is assigned a value but never used. ([view](https://sider.review/gh/repos/470071320/pulls/10#issue-6821393))

<details>

| Rule | Severity | Recommended |
| ---- | -------- | ----------- |
| `no-unused-vars` | `warn` | Yes |

References:
- <https://eslint.org/docs/rules/no-unused-vars>

You can [close this issue](https://sider.review/gh/repos/470071320/pulls/10#issue-6821393) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).

</details>

File: src/pages/jaehyuk/Login/LoginJaehyuk.js [hunk 4]
```diff
@@ -1,7 +1,90 @@
...
+  const isPassLogin = () => {};
```
Comment (ghost) [inline src/pages/jaehyuk/Login/LoginJaehyuk.js:15]: __[ESLint]__ 'isPassLogin' is assigned a value but never used. ([view](https://sider.review/gh/repos/470071320/pulls/10#issue-6821394))

<details>

| Rule | Severity | Recommended |
| ---- | -------- | ----------- |
| `no-unused-vars` | `warn` | Yes |

References:
- <https://eslint.org/docs/rules/no-unused-vars>

You can [close this issue](https://sider.review/gh/repos/470071320/pulls/10#issue-6821394) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).

</details>

File: src/pages/jaehyuk/Login/LoginJaehyuk.js [hunk 5]
```diff
@@ -1,7 +1,90 @@
...
+
+  const checkInput = () => {
+    setColor(email.indexOf('@') && password.length > 4 ? 'blue' : '#B2DFFC');
+  };
+  // const activateBtn = () => {
+  //   setColor('blue');
+  //   setIsActive('active');
+  //   setIsDisable(true);
+  // };
+  const deActivateBtn = () => {
```
Comment (ghost) [inline src/pages/jaehyuk/Login/LoginJaehyuk.js:35]: __[ESLint]__ 'deActivateBtn' is assigned a value but never used. ([view](https://sider.review/gh/repos/470071320/pulls/10#issue-6821395))

<details>

| Rule | Severity | Recommended |
| ---- | -------- | ----------- |
| `no-unused-vars` | `warn` | Yes |

References:
- <https://eslint.org/docs/rules/no-unused-vars>

You can [close this issue](https://sider.review/gh/repos/470071320/pulls/10#issue-6821395) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).

</details>

File: src/pages/jaehyuk/Main/MainJaehyuk.js [hunk 6]
```diff
@@ -1,7 +1,231 @@
-import React from 'react';
+import React, { useState } from 'react';
```
Comment (ghost) [inline src/pages/jaehyuk/Main/MainJaehyuk.js]: __[ESLint]__ 'useState' is defined but never used. ([view](https://sider.review/gh/repos/470071320/pulls/10#issue-6821396))

<details>

| Rule | Severity | Recommended |
| ---- | -------- | ----------- |
| `no-unused-vars` | `warn` | Yes |

References:
- <https://eslint.org/docs/rules/no-unused-vars>

You can [close this issue](https://sider.review/gh/repos/470071320/pulls/10#issue-6821396) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).

</details>

File: src/pages/jaehyuk/Main/MainJaehyuk.js [hunk 7]
```diff
@@ -1,7 +1,231 @@
...
+import { useNavigate } from 'react-router-dom';
```
Comment (ghost) [inline src/pages/jaehyuk/Main/MainJaehyuk.js:2]: __[ESLint]__ 'useNavigate' is defined but never used. ([view](https://sider.review/gh/repos/470071320/pulls/10#issue-6821397))

<details>

| Rule | Severity | Recommended |
| ---- | -------- | ----------- |
| `no-unused-vars` | `warn` | Yes |

References:
- <https://eslint.org/docs/rules/no-unused-vars>

You can [close this issue](https://sider.review/gh/repos/470071320/pulls/10#issue-6821397) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).

</details>

File: src/pages/jaehyuk/Components/Comments/input.comment.js [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+      }
+    }
+  };
+
+  return (
+    <>
+      <div className="commentBox">
+        <input
+          id="input-text"
+          name={props.inputName}
```
Comment (ghost) [inline src/pages/jaehyuk/Components/Comments/input.comment.js]: __[ESLint]__ Must use destructuring props assignment ([view](https://sider.review/gh/repos/470071320/pulls/10#issue-6821398))

<details>

| Rule | Severity | Recommended |
| ---- | -------- | ----------- |
| `react/destructuring-assignment` | `warn` | No |

References:
- <https://github.com/yannickcr/eslint-plugin-react/tree/master/docs/rules/destructuring-assignment.md>

You can [close this issue](https://sider.review/gh/repos/470071320/pulls/10#issue-6821398) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).

</details>

File: src/pages/jaehyuk/Components/Comments/input.comment.js [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+          placeholder={`${props.inputName}`.concat('..')}
```
Comment (ghost) [inline src/pages/jaehyuk/Components/Comments/input.comment.js]: __[ESLint]__ Must use destructuring props assignment ([view](https://sider.review/gh/repos/470071320/pulls/10#issue-6821399))

<details>

| Rule | Severity | Recommended |
| ---- | -------- | ----------- |
| `react/destructuring-assignment` | `warn` | No |

References:
- <https://github.com/yannickcr/eslint-plugin-react/tree/master/docs/rules/destructuring-assignment.md>

You can [close this issue](https://sider.review/gh/repos/470071320/pulls/10#issue-6821399) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).

</details>

File: src/pages/jaehyuk/Main/MainJaehyuk.js [hunk 10]
```diff
@@ -1,7 +1,231 @@
...
+                            mong_dory님 외 100명이 좋아합니다
+                          </span>
+                        </div>
+                      </div>
+                    </div>
+                    <div className="submit-form">
+                      <div id="input-comment">
+                        <div id="input-comment-view"></div>
+                        <div className="input-comment-text">
+                          {<InputComment inputName="댓글달기" />}
```
Comment (ghost) [inline src/pages/jaehyuk/Main/MainJaehyuk.js]: __[ESLint]__ Curly braces are unnecessary here. ([view](https://sider.review/gh/repos/470071320/pulls/10#issue-6821400))

<details>

| Rule | Severity | Recommended |
| ---- | -------- | ----------- |
| `react/jsx-curly-brace-presence` | `warn` | No |

References:
- <https://github.com/yannickcr/eslint-plugin-react/tree/master/docs/rules/jsx-curly-brace-presence.md>

You can [close this issue](https://sider.review/gh/repos/470071320/pulls/10#issue-6821400) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).

</details>

File: src/pages/jaehyuk/Main/MainJaehyuk.js [hunk 11]
```diff
@@ -1,7 +1,231 @@
...
+ * 2. validation comments
+ * 3. pass -> array.push(comments)
+ * 4. array.map() -> comments
+ */
 
 function Main() {
-  return <div> test </div>;
+  return (
+    <div className="root-container">
+      <Nav />
```
Comment (ghost) [inline src/pages/jaehyuk/Main/MainJaehyuk.js:32]: __[ESLint]__ 'Nav' is not defined. ([view](https://sider.review/gh/repos/470071320/pulls/10#issue-6821401))

<details>

| Rule | Severity | Recommended |
| ---- | -------- | ----------- |
| `react/jsx-no-undef` | `error` | Yes |

References:
- <https://github.com/yannickcr/eslint-plugin-react/tree/master/docs/rules/jsx-no-undef.md>

You can [close this issue](https://sider.review/gh/repos/470071320/pulls/10#issue-6821401) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).

</details>

File: src/pages/jaehyuk/Main/MainJaehyuk.js [hunk 12]
```diff
@@ -1,7 +1,231 @@
...
+      <div className="main">
+        <div className="blank"></div>
```
Comment (ghost) [inline src/pages/jaehyuk/Main/MainJaehyuk.js]: __[ESLint]__ Empty components are self-closing ([view](https://sider.review/gh/repos/470071320/pulls/10#issue-6821402))

<details>

| Rule | Severity | Recommended |
| ---- | -------- | ----------- |
| `react/self-closing-comp` | `warn` | No |

References:
- <https://github.com/yannickcr/eslint-plugin-react/tree/master/docs/rules/self-closing-comp.md>

You can [close this issue](https://sider.review/gh/repos/470071320/pulls/10#issue-6821402) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).

</details>

File: src/pages/jaehyuk/Main/MainJaehyuk.js [hunk 13]
```diff
@@ -1,7 +1,231 @@
...
+                        <div className="likes">
+                          <span className="font">
+                            mong_dory님 외 100명이 좋아합니다
+                          </span>
+                        </div>
+                      </div>
+                    </div>
+                    <div className="submit-form">
+                      <div id="input-comment">
+                        <div id="input-comment-view"></div>
```
Comment (ghost) [inline src/pages/jaehyuk/Main/MainJaehyuk.js]: __[ESLint]__ Empty components are self-closing ([view](https://sider.review/gh/repos/470071320/pulls/10#issue-6821403))

<details>

| Rule | Severity | Recommended |
| ---- | -------- | ----------- |
| `react/self-closing-comp` | `warn` | No |

References:
- <https://github.com/yannickcr/eslint-plugin-react/tree/master/docs/rules/self-closing-comp.md>

You can [close this issue](https://sider.review/gh/repos/470071320/pulls/10#issue-6821403) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).

</details>

File: src/components/Nav/Nav.scss [hunk 14]
```diff
@@ -1,17 +1,17 @@
-$border-color: 1px solid #DBDBDB;
+$border-color: 1px solid #dbdbdb;
```
Comment (sodalite1204) [inline src/components/Nav/Nav.scss:2]: 이미 variables.scss에 변수설정을 해 놓았기 때문에 재설정 할 필요가 없습니다.

File: src/components/Nav/Nav.scss [hunk 15]
```diff
@@ -1,17 +1,17 @@
...
   border-bottom: $border-color;
   width: 100%;
   display: flex;
   height: 60px;
-  background-color: #FFFFFF;
 
   .navBox {
     width: 65%;
     display: flex;
+    flex: 1;
```
Comment (sodalite1204) [inline src/components/Nav/Nav.scss:12]: mixin

File: src/pages/jaehyuk/Components/Comments/input.comment.js [hunk 16]
```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import './input.comment.scss';
+function InputComment(props) {
+  const [comment, setComment] = useState('');
+  const [commentArr, setcommentArr] = useState([]);
+  const onChange = event => setComment(event.target.value);
```
Comment (sodalite1204) [inline src/pages/jaehyuk/Components/Comments/input.comment.js]: 함수명!

File: src/pages/jaehyuk/Components/Comments/input.comment.js (hunk 9 above)
Comment (sodalite1204) [reply src/pages/jaehyuk/Components/Comments/input.comment.js]: 템플릿 리터럴

File: src/pages/jaehyuk/Components/Comments/input.comment.js [hunk 17]
```diff
@@ -0,0 +1,41 @@
...
+      <div className="commentBox">
+        <input
+          id="input-text"
+          name={props.inputName}
+          placeholder={`${props.inputName}`.concat('..')}
+          value={comment}
+          onChange={onChange}
+        />
+
+        <button id="btn-submit" onClick={onHandleSubmit}>
```
Comment (sodalite1204) [inline src/pages/jaehyuk/Components/Comments/input.comment.js]: id 사용
Camel Case / Kebab Case

File: src/pages/jaehyuk/Login/LoginJaehyuk.js [hunk 18]
```diff
@@ -1,7 +1,90 @@
...
+          <input
+            id="password-input-1"
+            type="password"
+            placeholder="비밀번호"
+            value={password}
+            onChange={handlePasswordInput}
+          />
+          <button
+            id="login"
+            style={{ backgroundColor: `${color}` }}
```
Comment (sodalite1204) [inline src/pages/jaehyuk/Login/LoginJaehyuk.js:87]: inline style

File: src/pages/jaehyuk/Main/MainJaehyuk.scss [hunk 19]
```diff
@@ -1 +1,376 @@
-// scss
\ No newline at end of file
+html,
```
Comment (sodalite1204) [inline src/pages/jaehyuk/Main/MainJaehyuk.scss:1]: html, body css
nesting

File: src/pages/jaehyuk/Main/MainJaehyuk.scss [hunk 20]
```diff
@@ -1 +1,376 @@
...
+  margin: 0px 10px 5px 10px;
+}
+
+.story-profile-pic {
+  margin: 5px 10px 0px 0px;
+}
+
+span {
+  display: block;
+}
```
Comment (sodalite1204) [inline src/pages/jaehyuk/Main/MainJaehyuk.scss:129]: 스타일 주는 순서

File: src/pages/jaehyuk/Components/Comments/InputComment.js [hunk 21]
```diff
@@ -0,0 +1,46 @@
...
+      }
+    }
+  };
+
+  let mockCommentArray = props.mockComment;
+  return (
+    <>
+      <div className="commentBox">
+        <input
+          id="input-text"
```
Comment (sodalite1204) [inline src/pages/jaehyuk/Components/Comments/InputComment.js:36]: id

File: src/pages/jaehyuk/Components/Comments/InputComment.js [hunk 22]
```diff
@@ -0,0 +1,46 @@
...
+          name={props.inputName}
+          placeholder={`${props.inputName}`.concat('..')}
```
[the remark this answers is not in the document]
Comment (sodalite1204) [reply src/pages/jaehyuk/Components/Comments/InputComment.js]: 템플릿 리테럴 사용

File: src/pages/jaehyuk/Login/LoginJaehyuk.js [hunk 23]
```diff
@@ -1,7 +1,119 @@
...
+import React, { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import './LoginJaehyuk.scss';
 
 function Login() {
-  return <div> test </div>;
+  const navigate = useNavigate();
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [isActive, setIsActive] = useState('deactive');
```
Comment (sodalite1204) [inline src/pages/jaehyuk/Login/LoginJaehyuk.js:9]: isActive

File: src/pages/jaehyuk/Login/LoginJaehyuk.js [hunk 24]
```diff
@@ -1,7 +1,119 @@
...
+
+  const onHandleIdInput = e => {
+    // setEmail {e.target.value}
+    setEmail(e.target.value);
+    checkInput();
+  };
+  const handlePasswordInput = e => {
+    setPassword(e.target.value);
+    checkInput();
+  };
```
Comment (sodalite1204) [inline src/pages/jaehyuk/Login/LoginJaehyuk.js:25]: 함수 호출
합치기

File: src/pages/jaehyuk/Login/LoginJaehyuk.js [hunk 25]
```diff
@@ -1,7 +1,119 @@
...
+
+  const checkInput = () => {
+    setColor(email.indexOf('@') && password.length > 4 ? 'blue' : '#B2DFFC');
```
Comment (sodalite1204) [inline src/pages/jaehyuk/Login/LoginJaehyuk.js:28]: 변수

File: src/pages/jaehyuk/Login/LoginJaehyuk.js [hunk 26]
```diff
@@ -1,7 +1,119 @@
...
 
 function Login() {
-  return <div> test </div>;
+  const navigate = useNavigate();
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [isActive, setIsActive] = useState('deactive');
+  const [isDisable, setIsDisable] = useState(true);
+  const [color, setColor] = useState('#1d84c9');
+  console.log(email);
```
Comment (sodalite1204) [inline src/pages/jaehyuk/Login/LoginJaehyuk.js:12]: console

File: src/pages/jaehyuk/Login/LoginJaehyuk.js [hunk 27]
```diff
@@ -1,7 +1,119 @@
...
+  };
+
+  const checkInput = () => {
+    setColor(email.indexOf('@') && password.length > 4 ? 'blue' : '#B2DFFC');
+  };
+  // const activateBtn = () => {
+  //   setColor('blue')
+  //   setIsActive('active')
+  //   setIsDisable(true)
+  // }
```
Comment (sodalite1204) [inline src/pages/jaehyuk/Login/LoginJaehyuk.js:34]: 주석

File: src/pages/jaehyuk/Login/LoginJaehyuk.js [hunk 28]
```diff
@@ -1,7 +1,119 @@
...
+    setIsActive('deactive');
+    setIsDisable(false);
+  };
+  // const navigate = useNavigate()
+  // const goToLogin = () => {
+  //   navigate('/jaehyuk/main')
+  // }
+  const onHandleSubmit = () => {
+    if (color === 'blue') {
+      navigate('/jaehyuk/main');
```
Comment (sodalite1204) [inline src/pages/jaehyuk/Login/LoginJaehyuk.js:45]: 조건

File: src/pages/jaehyuk/Login/LoginJaehyuk.js [hunk 29]
```diff
@@ -1,7 +1,119 @@
...
+      body: JSON.stringify({
+        name: '임재혁',
+        password: password,
+        email: email,
+        phone_number: '010-8949-4632',
+        address: '송파구',
+      }),
+    })
+      .then(res => res.json())
+      .then(data => console.log(data));
```
Comment (sodalite1204) [inline src/pages/jaehyuk/Login/LoginJaehyuk.js:59]: console

File: src/pages/jaehyuk/Login/LoginJaehyuk.js [hunk 30]
```diff
@@ -1,7 +1,119 @@
...
+//       name:'임재혁',
+//       password:'영문 소문자 영문 대문자 특수문자',
+//       email:'ljh95310@naver.com',
+//       phone_number:'010-8949-4632',
+//       address:'송파구'
+//     }),
+//   }
+//   .then((response) => response.json())
+//   .then((result) => console.log("결과: ", result))
+// }
```
Comment (sodalite1204) [inline src/pages/jaehyuk/Login/LoginJaehyuk.js:119]: 주석

File: src/pages/jaehyuk/Login/LoginJaehyuk.scss [hunk 31]
```diff
@@ -1 +1,92 @@
...
\ No newline at end of file
+html,
+body {
+  display: flex;
+  flex: 1;
+  height: 100%;
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+}
```
Comment (sodalite1204) [inline src/pages/jaehyuk/Login/LoginJaehyuk.scss:9]: html, body

File: src/pages/jaehyuk/Main/MainJaehyuk.js [hunk 32]
```diff
@@ -1,7 +1,243 @@
...
+                      <img
+                        alt=""
+                        className="icon"
+                        src="/images/Jaehyuk/jh-images/icon-image/comment.png"
+                      />
+                      <img
+                        alt=""
+                        className="icon"
+                        src="/images/Jaehyuk/jh-images/icon-image/direct-massage.png"
+                      />
```
Comment (sodalite1204) [inline src/pages/jaehyuk/Main/MainJaehyuk.js]: 반복 map

File: src/pages/jaehyuk/Main/MainJaehyuk.js [hunk 33]
```diff
@@ -1,7 +1,243 @@
...
+                  <img
+                    alt=""
+                    className="story-profile-pic"
+                    src="/images/Jaehyuk/jh-images/profiles/story-pic-1.jpeg"
+                  />
+                  <p>
+                    <span className="user-nickname">Jaehyukss</span>
+                    <span className="recommand-user">회원님을 위한 추천</span>
+                  </p>
+                </div>
```
Comment (sodalite1204) [inline src/pages/jaehyuk/Main/MainJaehyuk.js:84]: 반복 map

File: src/pages/jaehyuk/Main/MainJaehyuk.scss [hunk 34]
```diff
@@ -1 +1,375 @@
...
+
+.comment {
+  display: flex;
+}
+
+form {
+  display: flex;
+  justify-content: space-between;
+}
```
Comment (sodalite1204) [inline src/pages/jaehyuk/Main/MainJaehyuk.scss:197]: tag selector

File: src/pages/jaehyuk/Main/MainJaehyuk.scss [hunk 35]
```diff
@@ -1 +1,375 @@
...
+  margin: 0px 10px 5px 10px;
+}
+
+.story-profile-pic {
+  margin: 5px 10px 0px 0px;
+}
+
+span {
+  display: block;
+}
```
Comment (sodalite1204) [inline src/pages/jaehyuk/Main/MainJaehyuk.scss:129]: tag selector

File: src/pages/jaehyuk/Components/Comments/InputComment.js [hunk 36]
```diff
@@ -0,0 +1,46 @@
...
+    if (comment.length > 0) {
+      if (commentArr.length === 0) {
+        setcommentArr([comment]);
+      } else {
+        setcommentArr([...commentArr, comment]);
+      }
+    }
+  };
+
+  let mockCommentArray = props.mockComment;
```
Comment (sodalite1204) [inline src/pages/jaehyuk/Components/Comments/InputComment.js]: 변수

File: src/pages/jaehyuk/Main/MainJaehyuk.js [hunk 37]
```diff
@@ -1,7 +1,243 @@
...
+                        </div>
+                      </div>
+                    </div>
+                    <div className="submit-form">
+                      <div id="input-comment">
+                        <div id="input-comment-view"></div>
+                        <div className="input-comment-text">
+                          <InputComment
+                            mockComment={mockArray}
+                            inputName="댓글달기"
```
Comment (sodalite1204) [inline src/pages/jaehyuk/Main/MainJaehyuk.js]: props

**Pull request closed without merging**